### PR TITLE
feat: added branch descriptions

### LINKED ISSUE #67: [REQUEST] Mini description
**Is your feature request related to a problem? Please describe.**

This is not related to a bug

**Describe the feature you'd like to see implemented**

When you select a mod version it would be cool if you could get a description of what is special about that version. For example, for the FBW version, it could say an experimental version of the a32nx that adds a more realistic Flybywire system to the aircraft.

**Additional context**

Discord username: Awemeter#7919

### PR DESCRIPTION
**Pull request by ZigTag**
Fixes #67 

## Screenshots (if necessary)

image

## Additional context

Discord username (if different from GitHub):

### CODE REVIEW

**Comment by nistei** [comment]
Should maybe change the heading to `Version Info` since we call them mainline and experimental versions.

File: src/renderer/components/App/index.tsx [hunk 1]
```diff
@@ -158,6 +174,13 @@ function App() {
                             name: 'Custom FBW',
                             key: 'a32nx-fbw',
                             url: 'https://flybywiresim-packages.b-cdn.net/vmaster-cfbw/A32NX-master-cfbw.zip',
+                            description:
+                                <>
+                                    <p>
+                                        The Custom FBW version is exactly the same as Development but it contains our own Custom Fly-By-Wire system.
+                                        This branch is updated whenever the 'fbw' branch on Github is updated, which is around every 12 hours.
```
Comment (FoxtrotSierra6829) [inline src/renderer/components/App/index.tsx:181]: This branch -> this version

File: src/renderer/components/App/index.tsx [hunk 2]
```diff
@@ -175,6 +198,13 @@ function App() {
                             name: 'Custom FBW+AP',
                             key: 'a32nx-ap',
                             url: 'https://flybywiresim-packages.b-cdn.net/vmaster-cfbw-cap/A32NX-master-cfbw-cap.zip',
+                            description:
+                                <>
+                                    <p>
+                                        This is the exact same as Development but contains the Custom Fly-By-Wire and Custom Autopilot.
+                                        This branch is updated whenever the 'autopilot' branch on Github is updated, which is around every 12 hours.
```
Comment (FoxtrotSierra6829) [inline src/renderer/components/App/index.tsx:205]: This branch -> this version

**Review by FoxtrotSierra6829 — CHANGES_REQUESTED** [review]

**Comment by Benjozork** [comment]
I don't like the design of this, should maybe be a tooltip on the version ( with a ? icon or something)

File: src/renderer/components/App/index.tsx [hunk 3]
```diff
@@ -137,6 +138,13 @@ function App() {
                             name: 'Stable',
                             key: 'a32nx-stable',
                             url: 'https://flybywiresim-packages.b-cdn.net/stable/A32NX-stable.zip',
+                            description:
+                                <>
+                                    <p>
+                                        Stable is our variant that has the least bugs and best performance. This branch will not
```
Comment (FoxtrotSierra6829) [inline src/renderer/components/App/index.tsx:144]:
```suggestion
                                        Stable is our variant that has the least bugs and best performance. This version will not
```

Comment (FoxtrotSierra6829) [reply src/renderer/components/App/index.tsx:144]: Sorry, found another

**Review by FoxtrotSierra6829 — APPROVED** [review]

**Comment by FoxtrotSierra6829** [comment]
Other than the concern of @Benjozork, LGTM

**Pull request merged**